### DISCUSSION TITLE
gd works, gi does not, what am I missing

### DISCUSSION THREAD

**Discussion by laoshaw** [neoclide/coc.nvim#4804] [General]
with ctags I can ctrl-] to jump to implementation of the function, with coc.nvim I can only do gd to jump to definition, but gi does not work for me, what am I missing? do I need a compile_commands.json prebuilt at the topdir or a .clangd with all local directories added? Thanks.

**Comment by fannheyward** [comment]
Language server issue, `clangd` didn't returns result for `implementation` request.

**Comment by laoshaw** [comment]
this is for C, I am really surprised clangd can not find implementation of functions. thanks!
